format/macho/testdata/darwin_amd64/libbbb.a.fqtest

```diff
@@ -0,0 +1,33 @@
+$ fq -d macho v libbbb.a
```

Comment:
Could there be bytes before the ar archive starts? if not i wonder if both the ar and macho decoder will overlap now. Maybe is would make sense to add macho heuristics to the ar decoder instead? maybe even have SYMDEF as a sub format in some group and probe for in the ar decoder? But if we don't know yet maybe better to remove macho ar things for now and add it in separate PR?

Reply:
We can leave archive parsing out directly. I think it is cross platform. We can add it later on if there is demand. But I do not think so.

Reply:
👍